src/constants/style.scss

```diff
@@ -76,7 +76,7 @@ $board__side-panel-width: 64px;
 $header__height: 100px;
 $column__min-width: 300px;
 $column__max-width: 960px;
-$column__border-width: 12px;
+$column__border-width: 1.1vh;
```

Comment:
Not sure about that change, especially for mobile devices in either portrait or landscape mode. Did you play around with different settings or try to work with `min()` or `max()` here?

Reply:
Good point, `min()` or `max()` (or `vmin` / `vmax`) would be a good idea imo. However, I just noticed that when using `vw` or `vh` some borders are randomly not visible on some screen sizes, so I reverted the change for now.